source/Core/Utils.php

```diff
@@ -1069,17 +1069,28 @@ protected function _simpleRedirect($sUrl, $sHeaderCode)
     }
 
     /**
-     * Redirects to shop offline page
-     *
+     * Shows offline page
+     * @depricated please use showOfflinePage
      * @param int $iHeaderCode header code, default 302
      */
     public function redirectOffline($iHeaderCode = 302)
     {
-        $sUrl = $this->getConfig()->getShopUrl() . 'offline.html';
-        $this->redirect($sUrl, false, $iHeaderCode);
+        $this->showOfflinePage();
     }
 
     /**
+     * shows offline page
```

Comment:
Could you please add some information about the 500 header to the description of function.